gnome/gsettings-desktop-schemas/Portfile

```diff
@@ -5,6 +5,7 @@ PortGroup           meson 1.0
 
 name                gsettings-desktop-schemas
 version             3.38.0
+revision            1
```

Comment:
no need to increase the `revision` field as this is only a build dependency (see the relevant part of the [guide](https://guide.macports.org/#reference.keywords))